content/pages/resources.md

```diff
@@ -27,15 +27,29 @@ Here is a roundup of resources we commonly recommend:
 
 [Learn Python](https://www.learnpython.org/)
 
-[Codecademy](https://www.codecademy.com/learn/learn-python)
+[Codecademy](https://www.codecademy.com/learn/learn-python)  -  Codecademy offers a free beginner-friendly course in the basics of Python.
 
-[Free Code Camp](https://guide.freecodecamp.org/python/python-resources/)
+[Free Code Camp](https://guide.freecodecamp.org/python/python-resources/) provides an extensive list of Python tutorials.
 
 [Codesters](https://www.codesters.com/)
 
-[The Python Software Foundation Tutorial](https://docs.python.org/3/tutorial/)
+[The Python Software Foundation Tutorial](https://docs.python.org/3/tutorial/) - Python's official tutorial
 
-[First-Timers Only](https://www.firsttimersonly.com/)(.com)
+[First-Timers Only](https://www.firsttimersonly.com/)(.com) - This website is devoted to getting beginners involved in the Open Source community.
+
+[After Hours Programming](https://www.afterhoursprogramming.com/tutorial/python/python-overview/) - This tutorial covers language fundamentals and some web development concepts.
+
+[The Hitchhiker's Guide to Python](https://docs.python-guide.org/)  - This is a comprehensive guide covering best practices and uses of Python.
+
+[Invent Your Own Computer Games with Python 4th Edition](https://inventwithpython.com/invent4thed/) - This book teaches readers how to program in Python by making simple, classic games like Hangman and Tic Tac Toe.
+
+[Google's Python Class](https://developers.google.com/edu/python/?hl=en) -  This class has a slight barrier to entry but covers great topics like Lists, Dictionaries, and Regular Expressions
+
+[RealPython](https://realpython.com/) - This site has tons of tutorials, some free and some paid.  It covers a wide range of topics including language fundamentals, machine learning, testing, frameworks, and many more.
```

Comment:
Ditto my comment on Full Stack Python here. Ditto times like five!

Reply:
Also comprehensive.

Reply:
(Not to be moved to web dev section)

Reply:
This section is getting a bit long, so it might be time to add a section for these general resources that are more comprehensive or not just tutorials. But maybe that’s not necessary just yet. Let’s see how the other changes look first & then decide if what we have in this section has a clear-ish delineation.